src/ui/public/index_patterns/_get.js

```diff
@@ -21,7 +21,12 @@ export function IndexPatternsGetProvider(Private) {
       fields: [],
```

Comment:
Nice find. That seems like the right way to use it. I'd recommend updating to this approach and testing to make sure it works as intended @nreese